### PR TITLE
Update localization [05/17/23] 

### PR DESCRIPTION
**Pull request by vmapetr**
This PR updates localization for the following tasks:

- NodeTaskRunnerInstallerV0
- PublishBuildArtifactsV1
- AzureContainerAppsV0
- AzureContainerAppsV1
- MysqlDeploymentOnMachineGroupV1

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/PublishBuildArtifactsV1/task.loc.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
   "version": {
     "Major": 1,
     "Minor": 222,
-    "Patch": 1
+    "Patch": 2
```
Comment (KonstantinTyukalov) [inline Tasks/PublishBuildArtifactsV1/task.loc.json:16]: Probably, we should bump it up to 223 since today is Wednesday

**Review by KonstantinTyukalov — APPROVED** [review]

File: Tasks/PublishBuildArtifactsV1/task.loc.json (hunk 1 above)
Comment (vmapetr) [reply Tasks/PublishBuildArtifactsV1/task.loc.json:16]: Oh, missed it, thanks!

Comment (vmapetr) [reply Tasks/PublishBuildArtifactsV1/task.loc.json]: Fixed.

**Review by DmitriiBobreshev — APPROVED** [review]

**Review by LiliaSabitova — APPROVED** [review]

**Review by LiliaSabitova — APPROVED** [review]

**Review by kirill-ivlev — APPROVED** [review]

**Review by kirill-ivlev — APPROVED** [review]

**Pull request merged**